src/db/seeder.ts

```diff
@@ -56,6 +62,33 @@ export const init = async () => {
     accountID: bank.id,
     note: 'an income'
   });
+
+  const jack = await User.create({
+    username: 'jack',
+    email: 'jack.JJJ@gmail.com',
+    last_login: '2010/08/19',
+    date_login: '2011/09/10',
+    passwordhash: '89djheddkhdee33'
+  });
+
+  const tom = await User.create({
+    username: 'tom',
+    email: 'tom.c@Kmail.com',
+    last_login: '2011/10/19',
+    date_login: '2018/03/18',
+    passwordhash: '90d0dke888dje3j'
```

Comment:
in JS coding style, 

_ is not recommended


better to change name to lastLogin